### PR TITLE
move cmdLineAnnotator code to seperate module

### PR DESCRIPTION
**Pull request by bartcharbon**
#### Checklist
- [ ] Unit tests
- [ ] Updated testplan

### CODE REVIEW

File: molgenis-annotators-cmd/src/main/java/org/molgenis/annotation/cmd/CmdLineAnnotator.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.molgenis.data.annotation.cmd;
+package org.molgenis.annotation.cmd;
```
Comment (Mark-de-Haan) [inline molgenis-annotators-cmd/src/main/java/org/molgenis/annotation/cmd/CmdLineAnnotator.java]: /**
  * Build JAR file...............: mvn clean install -pl molgenis-data-annotators/ -am -DskipTests -P create-delivery
  * Run..........................: java -jar molgenis-data-annotators/target/CmdLineAnnotator.jar
  */

Change javadoc example commands to correct module paths

File: molgenis-data-annotators/pom.xml [hunk 2]
```diff
@@ -86,45 +86,6 @@
...
-                                    <goal>single</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
-
```
Comment (Mark-de-Haan) [inline molgenis-data-annotators/pom.xml]: This was removed but not added to the new POM. How to create runnable jar?

File: molgenis-annotators-cmd/src/main/java/org/molgenis/annotation/cmd/CmdLineAnnotator.java (hunk 1 above)
Comment (bartcharbon) [inline molgenis-annotators-cmd/src/main/java/org/molgenis/annotation/cmd/CmdLineAnnotator.java]: this is fixed, however the line note is not on the line of the problem, so github keeps showing this comment.

File: molgenis-data-annotators/pom.xml (hunk 2 above)
Comment (bartcharbon) [inline molgenis-data-annotators/pom.xml]: this is fixed, however the line note is not on the line of the problem, so github keeps showing this comment.

File: molgenis-annotators-cmd/pom.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <profile>
+            <id>create-delivery</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-assembly-plugin</artifactId>
+                        <configuration>
+                            <archive>
+                                <manifest>
+                                    <mainClass>org.molgenis.data.annotation.cmd.CmdLineAnnotator</mainClass>
```
Comment (Mark-de-Haan) [inline molgenis-annotators-cmd/pom.xml]: this should be **org.molgenis.annotation.cmd.CmdLineAnnotator**
Right now the Main class is not properly packaged in the Jar

**Pull request merged**